client/src/components/Menu/Menu.scss

```diff
@@ -224,10 +229,6 @@
 }
 
 .cms-menu__header {
-  position: relative !important; // sass-lint:disable-line no-important
-  top: auto !important; // sass-lint:disable-line no-important
-  // Required for JLayout
-  height: auto !important; // sass-lint:disable-line no-important
```

Comment:
Removed as no longer required as Jlayout was removed as far as I'm aware (and tested).